src/libtrx/game/anims/commands.c

```diff
@@ -0,0 +1,72 @@
+#include "benchmark.h"
+#include "game/anims.h"
+#include "game/gamebuf.h"
+
+static void M_ParseCommand(ANIM_COMMAND *command, const int16_t **data);
+
+static void M_ParseCommand(ANIM_COMMAND *const command, const int16_t **data)
+{
+    const int16_t *data_ptr = *data;
+    command->type = *data_ptr++;
+
+    switch (command->type) {
+    case AC_MOVE_ORIGIN: {
+        XYZ_16 *const pos = GameBuf_Alloc(sizeof(XYZ_16), GBUF_ANIM_COMMANDS);
+        command->data = (void *)pos;
```

Comment:
Let's move `command->data` assignments near `break;` for a cosmetic change – this will make the switch cases look more uniform.

Reply:
Done